docs/dg/dev/backend-development/data-manipulation/queue/queue.md

```diff
@@ -129,6 +129,25 @@ class SampleQueueMessageProcessorPlugin implements QueueMessageProcessorPluginIn
 ?>
 ```
 
+### Configuration for chunk size
```

Comment:
Is this section related to Queue message processor plugin?

Reply:
yes, it's a flexible configuration for the processor plugin that we enhanced in this ticket.